src/main/java/org/jenkinsci/plugins/workflow/graphanalysis/FlowScanningUtils.java

```diff
@@ -57,7 +51,7 @@ private FlowScanningUtils() {}
         return new Predicate<FlowNode>() {
             @Override
             public boolean apply(FlowNode input) {
-                return (input != null && input.getAction(actionClass) != null);
+                return (input != null && input.getPersistentAction(actionClass) != null);
```

Comment:
An incompatible change in case the `Action` was not a `PersistentAction`. If intentional, must at least change the method signature to require `PersistentAction` so that it becomes source-incompatible for any callers requesting another `Action` type.